frameworks/Java/light-java/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+sed -i 's|DATABASE_HOST|'"${DBHOST}"'|g' src/main/resources/config/mysql.json
+sed -i 's|DATABASE_HOST|'"${DBHOST}"'|g' src/main/resources/config/postgres.json
```

Comment:
We're trying to get everyone to move away from doing `sed` statements here and replacing all `localhost` urls with `TFB-database`. There are entries in the `etc/hosts` file